new-pipeline/src/dist.js

```diff
@@ -874,11 +874,14 @@ function displayHistograms(histogramsList, dates, useTable, cumulative, trim) {
     var description = histogramsList[0].histograms[0].description
     var metric = histogramsList[0].histograms[0].measure
     var channel = $("#channel-version").val().split("/")[0]
-    var desc = getDescriptionWithLink(metric, channel, description)
-
-    $('#dist-caption').html(desc);
+    var desc = getDescription(metric, channel, description);
+    var link = getDescriptionLink(metric, channel, description);
+    var div = document.getElementById('dist-caption-text');
```

Comment:
What is div used for?

Reply:
We can drop it. I used it initially for something. But now its not needed.